services/imagery/src/image-store.js

```diff
@@ -27,17 +27,17 @@ export default class ImageStore extends EventEmitter {
    * The rate at which images are added is also stored. Note that
    * this does not take in consideration the timestamp of the images.
    *
-   * A limit can be placed on the maximum number of images stored by
-   * setting the maxImages parameter. When the limit is reached,
+   * A limit can be placed on the maximum size of images stored by
+   * setting the maxStorage parameter. When the limit is reached,
```

Comment:
You should be absolutely clear on what the unit of `maxStorage` is supposed to be. I recommend megabytes.